www/src/pages/guidelines/color.js

```diff
@@ -25,15 +28,17 @@ import ColorModal from "../../components/guidelines/color/modal"
 
 import palette from "../../utils/guidelines/extend-palette-info"
 
-const LegacyColorIcon = ({ textColor }) => (
```

Comment:
🤦 🙄 LOL wondering what I was thinking there with `textColor` ;) — really appreciate your attention to these details! <3

Reply:
I can understand, I also added `box-sizing: border-box` to div even though it was declared using `*` selector :see_no_evil:
hahahahah